netshare/generate/model_generate.py

```diff
@@ -72,19 +73,25 @@ def generate_data(
 @ray.remote(scheduling_strategy="SPREAD", max_calls=1)
 def merge_attr(config_group: dict, configs: List[dict]) -> None:
     """
-    TODO: Can someone help me to doc this function?
+    This is the function to merge the attributes. The models trained on different
+    chunks of data will first generate attributes. Then, this function will be
+    invoked to merge the synthetic attributes. Given the merged attributes, models
+    will continue generating the time-series measurements.
+
+    :return: None
     """
     chunk0_idx = config_group["config_ids"][0]
     eval_root_folder = configs[chunk0_idx]["eval_root_folder"]
     attr_raw_npz_folder = os.path.join(eval_root_folder, "attr_raw")
-    word2vec_size = configs[chunk0_idx].get("word2vec_vecSize", 0)
-    pcap_interarrival = configs[chunk0_idx].get("timestamp") == "interarrival"
     num_chunks = len(config_group["config_ids"])
 
-    if not pcap_interarrival:
```

Comment:
The logic here is to find the column index which is a *flag* used to represent if the session (flow) start in this chunk.

Previously, the attribute schema is (5-tuple, [session_start_time], *flag*, ....).  Therefore, we need to have a if condition to get the correct index based on if the user specifies `interarrival`(`interarrival` means there is a session_start_time in attributes).

But, after the code refactoring, the schema is (5-tuple, *flag*, ...., , [session_start_time]). Therefore, we don't need this if condition logic anymore.

Reply:
Oh I'm surprised that I changed the schema. 
I'm so sorry for not noticing that!
I hope that the stronger E2E tests that Dorothy currently writing will help make sure that it won't happen again!
Thank you!!

Reply:
No wories. It's because `apply_timestamp_generation` called in `apply_distributed_chunk_logic` after `build_fields` being called in `setup_cross_chunks_data`

Reply:
Ohhh I see, wow, so sorry for it :(
If you prefer, we can also move the logic, or change the place where we put the new fields (i.e. the line `new_timeseries_list.insert(0, time_col)`)...
Amazing that you spot it 👑